pyembroidery/StringHelper.py

```diff
@@ -1,5 +1,5 @@
 def is_string(thing):
     try:
-        return isinstance(thing, basestring)
```

Comment:
I'd like to keep compatibility with python 2.7 if we can, to aid in the transition.  Maybe a try/except here to catch a NameError for basestring?

```
try:
    return isinstance(thing, basestring)
except NameError:
    return isinstance(thing, str)
```